frontend/src/App.jsx

```diff
@@ -61,75 +66,80 @@ function App() {
-    <Router>
-      <div className="App">
-        <header className="App-header">
-        {isLoggedIn && <Navbar onLogout={handleLogout} />} {/* Render Navbar if Logged In */}
-        {!isLoggedIn && <NavbarLoggedOut />} {/* Render if Logged Out */}
-        <p>
-          Social Predict: {backendData !== null ? backendData : 'Loading...'}
-        </p>
-          {/* Define Our Router */}
-          <Switch>
-            <Route exact path="/">
-              {!isLoggedIn ? <Login onLogin={handleLogin} /> : <Redirect to="/markets" />}
-            </Route>
-            <Route path="/profile">
-              {/* Render Profile only if not logged in, else redirect to "/" */}
-              {isLoggedIn ? <Profile /> : <Redirect to="/" />}
-            </Route>
-            {/* Render MarketDetails for individual marketId if either logged in or not*/}
-            <Route path="/markets/:marketId" component={MarketDetails} />
-            <Route path="/markets">
-              {/* Render Login only if not logged in */}
-              {!isLoggedIn && <Login onLogin={handleLogin} />}
-              {/* Add the Markets route */}
-              <Markets />
-            </Route>
-            <Route path="/polls">
-              {/* Render Login only if not logged in */}
-              {!isLoggedIn && <Login onLogin={handleLogin} />}
-              {/* Add the Polls route */}
-              <Polls />
-            </Route>
-            <Route path="/user/:username" component={User} />
-            <Route path="/notifications">
-              {/* Render Notifications only if not logged in, else redirect to "/" */}
-              {isLoggedIn ? <Notifications /> : <Redirect to="/" />}
-            </Route>
-            <Route path="/create">
-              {/* Render Create only if not logged in, else redirect to "/" */}
-              {isLoggedIn ? <Create /> : <Redirect to="/" />}
-            </Route>
-            <Route path="/about">
-              {/* Render Login only if not logged in */}
-              {!isLoggedIn && <Login onLogin={handleLogin} />}
-              <About />
-            </Route>
-            {/* Define other routes as needed */}
-          </Switch>
-        </header>
-        <Footer />
-      </div>
-    </Router>
+      <Router>
+        <div className='App'>
+          <header className='App-header'>
+            {isLoggedIn && <Navbar onLogout={handleLogout} />}{' '}
+            {/* Render Navbar if Logged In */}
+            {!isLoggedIn && <NavbarLoggedOut />} {/* Render if Logged Out */}
+            <p>
+              Social Predict:{' '}
+              {backendData !== null ? backendData : 'Loading...'}
+            </p>
+            {/* Define Our Router */}
+            <Switch>
+              <Route exact path='/'>
+                {!isLoggedIn ? (
+                  <Login onLogin={handleLogin} />
+                ) : (
+                  <Redirect to='/markets' />
+                )}
```

Comment:
As far as I can tell, the functionality is still the same -- why the new syntax? Does that have to do with it being .jsx?